### ISSUE TITLE
Show indication of who user is when hover over profile photo

### ISSUE THREAD

**Issue by patcon** [pol-is/issues#24]
Most people don't know the current profile photos of all their friends. It seems that pol.is team has taken great effort to show us people we know in the graph. But you can't easily see who's who unless you click through and open a bunch of tabs. I assume this is particularly difficult on mobile.

It would be great to have a tooltip, but the MVP for this would be if it were possible show the link in the bottom of the browser when we hover over the linked avatar image. This is default behaviour for browser links, but perhaps it is not so for iframes.

**Comment by mbjorkegren** [comment]
Good idea, and on mobile we could show the popover on click, and the popover would contain a button to open the profile in another tab.

**Comment by patcon** [comment]
cc: @derekhoward
